controller/golden_files_test.go

```diff
@@ -356,7 +356,17 @@ func TestGoldenReplaceTimes(t *testing.T) {
 		require.NoError(t, err)
 		require.Equal(t, expected, actual)
 	})
-
+	t.Run("date with IST timezone", func(t *testing.T) {
+		t.Parallel()
+		//given
+		str := `"last-modified": "Mon, 23 Apr 2018 00:00:00 IST",`
+		expected := `"last-modified": "Mon, 01 Jan 0001 00:00:00 GMT",`
```

Comment:
Can you maybe create a table-driven test for the three test dates: `"Thu, 15 Mar 2018 09:23:37 GMT"`, `"Fri, 13 Apr 2018 16:21:50 CEST"`, and `"Mon, 23 Apr 2018 00:00:00 IST"`? There's no need to copy the same test code over and over again because the expected result is always going to be `"Mon, 01 Jan 0001 00:00:00 GMT"`.